bloom/generators/common.py

```diff
@@ -114,6 +116,14 @@ def resolve_more_for_os(rosdep_key, view, installer, os_name, os_version):
                                              os_installers,
                                              default_os_installer)
     assert inst_key in os_installers
+    if 'BLOOM_SKIP_PIP' in environ and environ['BLOOM_SKIP_PIP'] == '1' and inst_key == 'pip':
+        # force set rule to null for pip, we can escape pip in
+        # L.99(def resolve_more_for_os(rosdep_key, view, installer,
+        # os_name, os_version):) in bloom/generators/common.py ?
+        warning("Key '{0}' resolved to '{1}' with installer '{2}' for os '{3}' '{4}', "
+                "with 'BLOOM_SKIP_PIP' environment raviable, we intentinally skip this rules"
```

Comment:
"but with the 'BLOOM_SKIP_PIP' environment variable set, this key is intentionally skipped."